test/components/form/batch-async-command.js

```diff
@@ -3,16 +3,15 @@
 import test from 'ava';
 import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
 import React from 'react';
-import { configure, shallow } from 'enzyme';
+import { configure, mount } from 'enzyme';
 
 import { BatchCommand } from '../../../app/src/js/components/BatchAsyncCommands/BatchAsyncCommands';
 
 configure({ adapter: new Adapter() });
 
-test('collect multiple errors', function (t) {
+test.cb('collect multiple errors', function (t) {
```

Comment:
since it's an AVA test and involves async ops, adding .cb is one of the ways to ensure that all async actions happen and finish before ending the test. I could've chosen to not use .cb and instead change function to `async` but that would entail making additional changes to the rest of the test file which I'm not very well-versed in